tools/openshift-ci/Dockerfile

```diff
@@ -23,10 +23,11 @@ RUN set -x && \
     CHROME_VERSION='116.0.5845.96' && \
     npx @puppeteer/browsers install chrome@${CHROME_VERSION} && \
     npx @puppeteer/browsers install chromedriver@${CHROME_VERSION} && \
+    find /chromedriver -type f -name chromedriver -exec cp {} /usr/local/bin/ \; && \
```

Comment:
Hi, you can `mv` it or `ln -s` it instead of duplicating the binary. If it is already executable, no need to `chmod` it later.

Reply:
`mv` can met selinux issue sometimes. And `chmod` to make sure it's executable.
Using links is a good idea. Updated to use hard links.

Reply:
inside containers, selinux has no effect afaict